.gitignore

```diff
@@ -81,5 +81,7 @@ packages/ckeditor5-watchdog/src/**/*.js
 packages/ckeditor5-widget/src/**/*.js
 packages/ckeditor5-word-count/src/**/*.js
 packages/*/src/**/*.d.ts
+packages/*/src/**/*.js.map
```

Comment:
Shouldn't we remove maps too? 

https://github.com/ckeditor/ckeditor5/blob/d88f9d31acbb4be6e3b10124c2c1ecf45bfed78a/scripts/release/clean.js#L39-L42